src/pe-csm-project-details/widget/pe-csm-project-details.u-option-schema.json

```diff
@@ -0,0 +1 @@
+[{"name":"title","section":"other","default_value":"Project details","label":"Title","type":"string"}]
```

Comment:
If you could please format this JSON so that  it renders well on the library portal and reads well. Thanks!